DataFormats/TauReco/interface/PFTau.h

```diff
@@ -127,23 +128,23 @@ class PFTau : public BaseTau {
 
     /// Retrieve the association of signal region gamma candidates into candidate PiZeros
     const std::vector<RecoTauPiZero>& signalPiZeroCandidates() const;
-    void setsignalPiZeroCandidates(const std::vector<RecoTauPiZero>&);
-    void setSignalPiZeroCandidatesRefs(const RecoTauPiZeroRefVector&);
+    void setsignalPiZeroCandidates(std::vector<RecoTauPiZero>);
```

Comment:
I changed these to pass by copy since internally the code will either do a copy or use move semantics. This change allows a more efficient call to this routine by using ```std::move()```.